docs/framework/guides/development-tools.md

```diff
@@ -35,6 +35,20 @@ or as a plain `<script>` tag in the HTML of your application:
 <script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
 ```
 
+#### Inspector as a bookmarklet
```

Comment:
I know it sounds a bit like a sub-section to "Importing the inspector" but heading level 4 is really deeeep in the structure. So maybe l3 looks better?